From b925b46415840a2a138356e73b4cac2e9b869ff4 Mon Sep 17 00:00:00 2001
From: Sumit Pundir <pundirsumit11@gmail.com>
Date: Mon, 16 Mar 2020 15:05:13 +0530
Subject: [PATCH] disable homepage

preference can be toggled within Settings->Home Page->on/off
---
 .../chrome/browser/partnercustomizations/HomepageManager.java   | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java b/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java
index 0ca6b3876bf4..fc7d7b56a302 100644
--- a/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java
+++ b/chrome/android/java/src/org/chromium/chrome/browser/partnercustomizations/HomepageManager.java
@@ -123,7 +123,7 @@ public class HomepageManager {
      * @see #isHomepageEnabled
      */
     public boolean getPrefHomepageEnabled() {
-        return mSharedPreferences.getBoolean(PREF_HOMEPAGE_ENABLED, true);
+        return mSharedPreferences.getBoolean(PREF_HOMEPAGE_ENABLED, false);
     }
 
     /**
-- 
2.20.1

